docs/sources/shared/upgrade/upgrade-common-tasks.md

```diff
@@ -119,7 +119,7 @@ To upgrade Grafana installed on Windows, complete the following steps:
 
 To upgrade Grafana installed on Mac, complete the following steps:
 
-1. In your current installation of Grafana, save your custom configuration changes to a file named `<grafana_install_dir>/conf/custom.ini`.
+1. In your current installation of Grafana, save your custom configuration changes to the custom configuration file, `custom.ini` or `grafana.ini`.
```

Comment:
I'm confused at this comment. This whole section is about how to update the custom files for each OS. I don't understand what flow is being broken. I've updated based on @ppcano's feedback about where the custom files are found based on the Configure Grafana docs. If the information about where the custom files are found isn't correct, then that needs to be updated by someone who knows this for sure, but this section isn't about the default files.

The reason we have this content repeated in sections is so that someone can go straight to their OS and have the information there. People don't always necessarily read the information at the top of a page and we've set up this content this way to facilitate it for the user (vs just writing it once ourselves and then never again).

Reply:
> Reference example: see NOTE on this page beginning:
https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#configure-grafana

@usmangt , that doc page says:

> Depending on your OS, your custom configuration file is either the $WORKING_DIR/conf/defaults.ini file or 


This seems wrong, doesn't it?

I've created a [PR](https://github.com/grafana/grafana/pull/83169) to reference `custom.ini`. Please, check this out.